docs/proposals/algorithms/lifelong-learning/Real-Time IoT Perception Systems Based on Edge-Cloud Collaboration with Large Foundation Models.md

```diff
@@ -0,0 +1,119 @@
+
+3. Deployment on Real Edge-Cloud Environment Platform:
+
+      Deploy the edge-cloud collaboration real-time IoT perception system developed based on Kubeedge/Sedna to **real edge-cloud environments** composed of devices such as **Nvidia Jetson Nano** and **Raspberry Pi**, and test the system's actual performance (latency, communication overhead, resource utilization, and other metrics).
+
+## 2 Proposal
+
+This project mainly focuses on semantic segmentation tasks in the AIoT environment, deploying **SAM** as the large model in the cloud and RFNet as the small model at the edge, and collaborating through **cloud-edge collaborative inference**.
+
+### 2.1 Introduction for Segment Anything Model
+
+The **Segment Anything Model ([SAM](https://segment-anything.com/)) **is a model designed for promptable segmentation, which is a task in image segmentation. SAM consists of three components: an image encoder, a prompt encoder, and a mask decoder. The image encoder computes an image embedding, the prompt encoder embeds prompts, and the mask decoder predicts segmentation masks by combining the information from the image and prompt encoders. SAM is trained to be promptable, meaning it can transfer its segmentation capabilities to new image distributions and tasks through prompt engineering. It is designed to be flexible, real-time, and ambiguity-aware. SAM has been evaluated on various tasks and has shown impressive zero-shot performance, often comparable to or better than fully supervised models.
+
+![sam](images/sam.png)
+
+SAM is designed to perform promptable segmentation, which means it can generate segmentation masks from a wide range of prompts. This flexibility allows SAM to be used for various segmentation tasks without the need for extensive retraining. SAM has the ability to transfer its segmentation capabilities to new image distributions and tasks without the need for additional training. This zero-shot transfer capability makes SAM highly adaptable and useful in scenarios where labeled data for specific tasks may be limited.
+
+### 2.2 Introduction for Cloud-Edge Collaborative Inference
+
+**Cloud-edge collaborative inference**, as an emerging technology, combines the processing capabilities of the cloud and edge devices to handle complex tasks more efficiently. The core idea of cloud-edge collaborative inference is to deploy large-scale models in the cloud, leveraging the powerful computing resources to handle complex tasks and challenging samples. On the other hand, deploying small-scale models on edge devices can meet the requirements for processing simple samples. In this architecture, edge devices only need to transmit sample data and communicate with the cloud, without bearing a large amount of computation tasks, greatly saving energy and computational resources.
+
+<img src="images/云边协同改进.png" alt="云边协同改进" style="zoom: 25%;" />
```

Comment:
Good to learn the joint inference with SAM and the ''edge model'' on the cloud. It fixed the problem that SAM does not output the inference class but only the segmentation result. But it is weird to see an "edge model" on the cloud. You might want to change the "edge model" to a "small model"?

Reply:
> Good to learn the joint inference with SAM and the ''edge model'' on the cloud. It fixed the problem that SAM does not output the inference class but only the segmentation result. But it is weird to see an "edge model" on the cloud. You might want to change the "edge model" to a "small model"?

I have changed the "edge model" to "RFNet model".